intrinsics/intrinsics_test.go

```diff
@@ -137,6 +138,11 @@ var _ = Describe("AWS CloudFormation intrinsic function processing", func() {
 			Expect(properties["Handler"]).To(Equal("method.test-parameter-value."))
 		})
 
+		// GetAtt isn't implemented today
+		It("should resolve a resource attribute within a !Sub", func() {
```

Comment:
I would change the test expectation to "should _not_ resolve a resource..". But this is fine too.